api/src/main/java/net/kyori/adventure/text/LinearComponents.java

```diff
@@ -39,6 +39,16 @@ private LinearComponents() {
   /**
    * Creates a component "linearly".
```

Comment:
A utility class that allows {@link Component components} to be created where {@link Style styles} can be specfied inline.

Reply:
Yeah -- it's not all that helpful to have a summary that just says the same words that are in the class name in a different order.